js/stex.js

```diff
@@ -2467,19 +2468,21 @@ module.exports = class stex extends Exchange {
         //     }
         //
         const data = this.safeValue (response, 'data', []);
-        const withdrawFees = {};
-        const depositFees = {};
+        const result = {};
```

Comment:
@AlgoQ once again I don't think we need this extra call since we have currencies loaded
![image](https://user-images.githubusercontent.com/43336371/198822885-4f5a01b5-a256-415a-8418-1d005220784d.png)